backend/test/mastodon.spec.ts

```diff
@@ -198,6 +198,52 @@ describe('Mastodon APIs', () => {
 				assert.equal(enrichStatus(`@!@£${link}!!!`, []), `<p>@!@£<a href="${link}">${urlDisplayText}</a>!!!</p>`)
 			})
 		})
+
+		test('convert tags to HTML', async () => {
+			const tagsToTest = [
+				{
+					tag: '#test',
+					expectedTagAnchor: '<a href="/tags/test" class="status-link hashtag">#test</a>',
+				},
+				{
+					tag: '#123_joke_123',
+					expectedTagAnchor: '<a href="/tags/123_joke_123" class="status-link hashtag">#123_joke_123</a>',
+				},
+				{
+					tag: '#_123',
+					expectedTagAnchor: '<a href="/tags/_123" class="status-link hashtag">#_123</a>',
+				},
+				{
+					tag: '#example:',
+					expectedTagAnchor: '<a href="/tags/example" class="status-link hashtag">#example</a>:',
+				},
+				{
+					// IS THIS OK?
+					tag: '#tagA#tagB',
```

Comment:
@xtuc what do you think? is this ok? should we recognize both tags? or should this just fail to recognize any?

Reply:
Tried with Mastodon and they only convert the first one into a link:
![image](https://user-images.githubusercontent.com/1493671/221923313-35a722b5-2096-4265-94e1-4184a216ec0e.png)


I think converting both to links is fine too.